mkosi

```diff
@@ -912,12 +915,105 @@ SigLevel    = Required DatabaseOptional
 
     enable_networkd(workspace)
 
+@complete_step('Installing openSUSE')
+def install_opensuse(args, workspace, run_build_script):
+    opensuse_main_repository_map = {
+        "tumbleweed" : "http://download.opensuse.org/%s/repo/oss/",
+        "42"         : "http://download.opensuse.org/distribution/leap/%s/repo/oss/",
+        "13"         : "http://download.opensuse.org/distribution/%s/repo/oss/",
+    }
+
+    opensuse_update_repository_map = {
+        "tumbleweed" : "http://download.opensuse.org/update/%s/",
+        "42"         : "http://download.opensuse.org/update/leap/%s/oss/",
+        "13"         : "http://download.opensuse.org/update/%s/",
+    }
```

Comment:
hmm, any chance this can be done more future-proof? i.e. if "14" appears that we don't have to update mkosi?